numba/core/decorators.py

```diff
@@ -11,6 +11,7 @@
 from numba.core.errors import DeprecationError, NumbaDeprecationWarning
 from numba.stencils.stencil import stencil
 from numba.core import config, sigutils, registry, cpu_dispatcher
+from numba.dppl import gpu_dispatcher
```

Comment:
Since we are using Sycl and the device can be any Sycl device not just a GPU. Maybe rename `gpu_dispatcher` to `sycl_device_dispatcher`